packages/p2p/src/components/my-profile/block-user/block-user-table/block-user-row.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Do we need multiple `block-user` folders in different places? Can't we just keep everything related to `block-user` inside `src/components/bock-user`?

Reply:
Initially we had planned to keep it in one separate folder 'block-user' but since the block user table only exists within my profile, we came to the conclusion that there should be a separate folder called 'block-user' and within that folder, another folder with block-user-table contents should also be inside it